doc/getting_started.md

```diff
@@ -503,13 +502,35 @@ Capturing: 100%|█████████████████████
 Created plot with 10 traces: ~/ot-sca/cw/cw305/projects/sample_traces_aes.html
 ```
 
+Following command may be used to capture AES traces in batch mode: 
+```console
+$ cd cw/cw305
+$ ./capture.py --cfg-file capture_aes.yaml capture aes-random-batch --num-traces 100 --plot-traces 10
+```
+
+By default, this will capture 5000 traces which should be sufficient to make
+sure the setup is configured and working as expected before starting a long
+running capture. The plot should look very similar to the one of the
+non-batched AES capture.
+
+To capture e.g. 2 Mio traces, open the config file `capture_aes.yaml` and change
+`capture.num_traces` to `2000000`. Also, it is recommended to set
+`plot_capture.show` to `false` as the routine checking the ADC gain setting
+(as part of the plot generation) can take a long time for high number of traces.
+
```

Comment:
Both these points are no longer required. The ADC outputs are now checked within the capture loop and independent of the plot setting. And thanks to your work, one can just provide a different `num_traces` argument using `--num-traces 2000000` :-)